modules/ui/osmose_editor.js

```diff
@@ -0,0 +1,170 @@
+import { dispatch as d3_dispatch } from 'd3-dispatch';
+
+import { t } from '../util/locale';
+import { services } from '../services';
+import { modeBrowse } from '../modes/browse';
+import { svgIcon } from '../svg/icon';
+
+import { uiOsmoseDetails } from './osmose_details';
+import { uiOsmoseHeader } from './osmose_header';
+import { uiQuickLinks } from './quick_links';
+import { uiTooltipHtml } from './tooltipHtml';
+
+import { utilRebind } from '../util';
+
+
+export function uiOsmoseEditor(context) {
```

Comment:
Is there any link format we can use for a "View on Osmose" button in the footer?

Reply:
There's the following information page:

```
https://osmose.openstreetmap.fr/error/e2a7b8fa-3ae8-c698-e488-7363adc27577
```

Alternatively we could link to the actual map:

```
https://osmose.openstreetmap.fr/map/?item=5070&zoom=17&lat=58.2055999&lon=-6.8280015
```

Reply:
Hmm the raw page isn't pretty but the map page doesn't show the expanded issue so let's go with the first one anyway.

Reply:
Added this in af0e37f7c56cc2adfd11bcef72fce8756a43f749